### PR TITLE
Cookbook for enabling Umpire logging

### PR DESCRIPTION
**Pull request by mcfadden8**

### CODE REVIEW

**Comment by davidbeckingsale** [comment]
I think this is okay for now, but in general, I don't think features like this should be presented as recipes. It's a fundamental option that applies to the whole library, and recipes were designed to show how to compose Umpire objects for complex use cases.

**Comment by mcfadden8** [comment]
I agree that recipes don't seem lik the right place to put this.  I'm happy to place it somewhere else.  Do you have any suggestions?

**Comment by mcfadden8** [comment]
What do you think of this @davidbeckingsale?

File: docs/sphinx/tutorial/replay.rst [hunk 1]
```diff
@@ -4,21 +4,6 @@
 Replay
 ======
 
-Umpire provides a lightweight replay capability that can be used to investigate
```
Comment (davidbeckingsale) [inline docs/sphinx/tutorial/replay.rst]: This paragraph can stay here.

Comment (davidbeckingsale) [reply docs/sphinx/tutorial/replay.rst]: But you don't want the bit about the list of events.

**Pull request merged**